google-cloud-firestore/lib/google/cloud/firestore.rb

```diff
@@ -390,6 +390,52 @@ module Cloud
     # user_ref.update({ nested_field_path => "Pasta" })
     # ```
     #
+    # ### Listening for changes
+    #
+    # You can listen to a document reference or a collection reference/query for
+    # changes. A document or query snapshot will be yielded immediately, and
```

Comment:
I think what we are trying to say here is that when you start listening, you get a snapshot of things as they are right now, and then you will get snapshots of changes as they happen.

For example, when you start listening to a document that isn't being changed, you will always get a snapshot of the document, but you won't get second snapshot because there are no changes being made.

Reply:
@schmidt-sebastian We adapted that documentation from [Get realtime updates with Cloud Firestore](https://firebase.google.com/docs/firestore/query-data/listen):

> You can listen to a document with the onSnapshot() method. An initial call using the callback you provide creates a document snapshot immediately with the current contents of the single document. Then, each time the contents change, another call updates the document snapshot.

If it is incorrect, let us know and we will remove it.

Reply:
How about replacing

> A document or query snapshot will be yielded immediately, and

with:

> The current document snapshot or query results snapshot will be yielded first, and

Reply:
Sounds good.